style.css

```diff
@@ -405,3 +448,229 @@ textarea::placeholder {
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+    text-decoration: none;
+    justify-content: space-between;
+  }
+
+  #hamburger {
+    display: none;
+  }
+
+  .headline {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    background-image: url("./assets/images/header-bg.png");
+    background-repeat: no-repeat;
+    background-position: 20px 0;
+    height: 825px;
+    width: 1585px;
+  }
+
+  #headline-paragraph,
+  #headline-text {
+    width: 920px;
+  }
+
+  #heading-social,
+  #icons {
+    align-self: flex-start;
+    position: relative;
+    left: 308px;
+  }
+
+  .break {
+    display: none;
+  }
+
+  .heading-social {
+    margin-left: 25px;
+    padding-left: 0;
+  }
+
+  .icons {
+    margin-left: 0;
+  }
+
+  .projects-section {
+    display: none;
+  }
+
+  .projects-section-desktop {
+    display: grid;
+    grid-template-rows: repeat(4, 1fr);
+    grid-template-columns: repeat(1fr);
+    place-items: center;
+    margin-top: 100px;
+    gap: 50px;
+    margin-left: 190px;
+  }
+
+  .projects-section-desktop .card {
+    display: flex;
+    flex-direction: row;
+    border: 1px solid #1e315f;
+    padding: 24px;
+    height: 496px;
+    width: 1156px;
+    gap: 12px;
+  }
+
+  .projects-section-desktop .card-image {
+    width: 100%;
+    height: 100%;
+  }
+
+  .projects-section-desktop .tonic-area {
+    color: #344563;
+    font-size: 2vw;
+    margin-left: 24px;
+  }
+
+  .projects-section-desktop .cards-details {
+    color: #091e42;
+  }
+
+  .projects-section-desktop .highlight-text {
+    margin: 10px 10px 10px 0;
+  }
+
+  .projects-section-desktop .countertext {
+    margin: 10px;
+  }
+
+  .projects-section-desktop .year {
+    margin-left: 10px;
+  }
+
+  .projects-section-desktop .card-description {
+    margin-left: 10px;
+  }
+
+  .projects-section-desktop .skills {
+    margin: 20px 10px 30px 0;
+  }
+
+  #project2 {
+    flex-direction: row-reverse !important;
+  }
+
+  #project4 {
+    flex-direction: row-reverse;
+  }
+
+  .about {
+    display: flex;
+    flex-direction: row;
+    margin: 50px 0 0 0;
+    width: 1585px;
+  }
+
+  .about-mobile {
+    margin-top: -100px;
+    margin-bottom: 50px;
+    width: 50%;
+    margin-left: 100px;
+  }
+
+  .about-lang {
+    margin-top: -10%;
+    margin-bottom: 80px;
+    font-size: 2vw;
+    width: 50%;
+  }
+
+  .lang {
+    margin-top: 12px;
+  }
+
+  .icons-lang {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .icons-lang li {
+    align-items: flex-start;
+    margin-top: 0;
+    margin-right: 5px;
+  }
+
+  .li-icons {
+    display: flex;
+    flex-direction: column;
+    width: 118px;
+    height: 100px;
+  }
+
+  .Separator {
+    width: 80%;
+  }
+
+  .contact-page-container {
+    background-image: url("./assets/images/contact\ form\ background\ shapes@2x.png");
+    width: 1585px;
```

Comment:
- [ ] As I mentioned the reasons earlier, kindly consider removing the exact widths on your containers.